Dockerfile

```
@@ -0,0 +1,54 @@
+FROM nginx:1.21.0
+
+
+# Install necessary packages 
+RUN apt-get update -y && apt-get install -y apt-transport-https ca-certificates wget \
+    gnupg2 lsb-release sl mlocate openssl
```

```suggestion
    gnupg2 lsb-release sl mlocate openssl && rm -rf /var/lib/apt/lists/*
```

Additionally, it seems to me `sl` and `mlocate` could be dropped. Should be tested though.